core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -440,7 +439,7 @@ export class LoggedOutUser extends BaseUser {
     if (this.isViewportAtMobileWidth()) {
       await this.clickOn(mobileNavbarOpenSidebarButton);
       await this.clickButtonToNavigateToNewPage(
-        mobileSidebarBasicMathematicsButton,
+        mobileSidebarClassroomButton,
         'Basic Mathematics button in the Learn Menu on navbar',
```

Comment:
@AFZL210 Please update the name of the button in this line as well, as you did with the constant name just above.

Reply:
Done.

Reply:
Also, please see line 444 and 452 (of the left side of the diff), those are the names of the button that the function is receiving as argument. You may want to update them as well as it's no longer just a mathematics classroom button.

Reply:
@Akhilesh-max This ```clickBasicMathematicsButtonInLearnMenuOnNavbar``` function is not used anywhere should I just remove it?

Reply:
If it is not used anywhere, it should be OK to remove it.
Thansk!

Reply:
Done.